src/warm/WarmCore.cpp

```diff
@@ -836,7 +836,7 @@ static PlatformResult ThreadAdvertisementAction(ActionType inAction, bool inActi
 {
     Inet::IPPrefix prefix;
 
-    MakePrefix(inGlobalId, 0, kGlobalULAPrefixLength, prefix);
+    MakePrefix(inGlobalId, kWeaveSubnetId_ThreadMesh, kThreadULAAddressPrefixLength, prefix);
```

Comment:
Now I understand your comment about packets being dropped when prefixlen != 64. This appears to be a bug in Warm that has gone unnoticed for a long time.